src/components/App.tsx

```diff
@@ -913,6 +914,11 @@ class App extends React.Component<any, AppState> {
         }
       };
 
+      const socketIOClientLib: any = await import(
+        /* webpackChunkName: "socketIoClient.min" */ "socket.io-client"
+      );
+      const socketIOClient = socketIOClientLib.default;
```

Comment:
```suggestion
      const { default: socketIOClient } = await import(
        /* webpackChunkName: "socketIoClient.min" */ "socket.io-client"
      );
```

what's the point of the webpack override btw?

Reply:
Do you mean the webpack chunk name ? 
It's just so that the named bundle is created instead of `[randomId].hash.js` and this gives an idea which chunk is taking what size.

Reply:
Doesn't help that much because webpack apparently splits it into 2 bundles for some reason, and the named chunk is just some thin wrapper:

![image](https://user-images.githubusercontent.com/5153846/82725967-2aed7080-9ce1-11ea-86dc-cd4e869a994e.png)

Reply:
I didn't understand clearly If you meant dynamic import is not helping or the webpack chunk name. So Dynamic Import has helped as now the above two bundles will be loaded when needed and yes I think react-scripts might be doing something internally which is again splitting this into two chunks. But if it's not causing any issue it would be better to keep. it as it might help if we later add our own webpack config to create umd bundle.
But again I am ok with either.
Before

<img width="458" alt="Screenshot 2020-05-23 at 2 44 44 PM" src="https://user-images.githubusercontent.com/11256141/82726792-140c4580-9d04-11ea-89e8-69895e6ce09f.png">

After

<img width="546" alt="Screenshot 2020-05-23 at 2 44 49 PM" src="https://user-images.githubusercontent.com/11256141/82726805-225a6180-9d04-11ea-999a-9fd3a0d08909.png">

Reply:
I meant the renaming, because it might confuse even more than if we didn't have it (you'll be left wondering what are unnamed chunks being loaded, but won't consider it may in fact be related to the named chunks). But I guess we can keep it.

Reply:
Yes but this will actually help for the umd build as we might want to limit the number of chunks in that case and this chunk name helps there. 

<img width="840" alt="Screenshot 2020-05-23 at 3 00 57 PM" src="https://user-images.githubusercontent.com/11256141/82727156-5cc4fe00-9d06-11ea-8901-e5768b10a8ab.png">

Since here we don't actually need to specify the mac chunks so webpack again is splitting the socket into chunks.